client/packages/glsp-sprotty/src/features/model-access/di.config.ts

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2018 EclipseSource
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * 	Philip Langer - initial API and implementation
+ ******************************************************************************/
+
+import { ContainerModule } from "inversify";
+import { TYPES } from "sprotty/lib";
+import { GLSP_TYPES } from "../../types";
+import { IModelAccess, ModelProvidingCommandStack } from "./model-access";
+
+const modelAccessModule = new ContainerModule((bind, unbind, isBound, rebind) => {
+    rebind(TYPES.ICommandStack).to(ModelProvidingCommandStack).inSingletonScope();
```

Comment:
This is problematic when used in combination with  the `ObservableCommandStack`.  Currently the visual feeback of resize handles gets lost on server update due to this rebind.
I guess we should introduce a base GLSPCommandStack that is both `Observable` and `ModelProviding` and bind Types.ICommandStack to this GLSPCommandStack

Reply:
Nice catch! I'll turn our command stack into `GLSPCommandStack` that implements both `IModelAccess` and a new interface `IModelUpdateNotifier`. I'll further register it as a service for both interfaces, so that the clients of `IModelUpdateNotifier` actually don't have to know whether it is implemented by the command stack or anything else.